Iss140: Update default values of Seurat runs (#142)
CCBR/SINCLAIR: CHANGELOG.md

- Before launching the pipeline run:
  - The nextflow preview is printed. (#117, @kelly-sovacool)
  - The nextflow parameters are validated. (#127, @kelly-sovacool)

> - Default parameters for variables to regress is "None", species is "hg38", and filtering method is "miqc" (#142, @wong-nw)

Reviewer: ```suggestion
- Default parameters changed: (#142, @wong-nw)
  - variables to regress is `null`
  - species is "hg38"
  - filtering method is "miqc"
```
